projects/rero/ng-core/src/lib/record/search/aggregation/list-filters/list-filters.component.html

```diff
@@ -0,0 +1,8 @@
+<ul *ngIf="filters?.length > 0" class="pl-0">
+    <li id="filter" *ngFor="let filter of filters" class="d-inline list-unstyled pr-2">
```

Comment:
use an `id` attribute with a `*ngFor` should be avoid. An ID **must** be unique into the DOM. If we create multiple `li`, the same `id` cill be used. 

Additionnally using a `id` attribute on this tag seems not required at all except for the `spec.ts`.
Two solutions :
* either use an `id='filters'` attributes on the `ul` tag. Selectror changes to `ul#filters li > button` (to be tested)
* either use an additional `class` (ex:'filter-value') on each `li`. Selectror changes to `li.filter-value > button`(to be tested)